<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2023)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="window_size" value="30" />
    <node pkg="assignment_2_2023" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2023" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2023" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment_2_2023" type="client.py" name="client_node" output="screen" launch-prefix="lxterminal -e"/>
    <node pkg="assignment_2_2023" type="goal_service.py" name="goal_service_node"/>
    <node pkg="assignment_2_2023" type="info_service.py" name="info_service_node"/>
</launch>
